src/WebJobs.Extensions.DurableTask/LifeCycleNotificationHelper.cs

```diff
@@ -210,7 +210,7 @@ public void SetHttpMessageHandler(HttpMessageHandler handler)
                         FunctionName = functionName,
                         InstanceId = instanceId,
                         Reason = reason,
-                        EventType = orchestrationRuntimeStatus,
+                        RuntimeStatus = $"{orchestrationRuntimeStatus}",
```

Comment:
`orchestrationRuntimeStatus.ToString()` is better from a performance perspective.

Reply:
Thanks, I've done.